src/components/pages/InstructorNewsFeed/NewsContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import {connect} from 'react-redux';
+import '../../../styles/index.less';
+import {getNewsFeeds} from '../../../redux/actions/instructorActions';
+function NewsContainer(props) {
+  const { setPostId, setPostOptions, newsfeed,dispatch }=props;
```

Comment:
Add a space between newsfeed and dispatch

Reply:
this one is done will push after fixing rest